x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/service.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ExceptionListItemSchema,
+  FoundExceptionListItemSchema,
+} from '@kbn/securitysolution-io-ts-list-types';
+import { ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID } from '@kbn/securitysolution-list-constants';
+import { HttpStart } from 'kibana/public';
+import { EXCEPTION_LIST_ITEM_URL, EXCEPTION_LIST_URL } from '../event_filters/constants';
+import { HOST_ISOLATION_EXCEPTIONS_LIST } from './constants';
+
+async function createHostIsolationExceptionList(http: HttpStart) {
+  try {
+    await http.post<ExceptionListItemSchema>(EXCEPTION_LIST_URL, {
+      body: JSON.stringify(HOST_ISOLATION_EXCEPTIONS_LIST),
+    });
+  } catch (err) {
+    // Ignore 409 errors. List already created
+    if (err.response?.status !== 409) {
+      throw err;
+    }
+  }
+}
+
+let listExistsPromise: Promise<void>;
+async function ensureHostIsolationExceptionsListExists(http: HttpStart) {
+  if (!listExistsPromise) {
+    listExistsPromise = createHostIsolationExceptionList(http);
+  }
+  await listExistsPromise;
+}
+
+export async function getHostIsolationExceptionItems({
+  http,
+  perPage,
+  page,
+  sortField,
+  sortOrder,
+  filter,
+}: {
+  http: HttpStart;
+  page?: number;
+  perPage?: number;
+  sortField?: keyof ExceptionListItemSchema;
+  sortOrder?: 'asc' | 'desc';
```

Comment:
nit: We have a `SortOrder` type that should be the same as re-doing this here but if you look across the code base the `asc | desc` has been retyped so many times you can pull from any number of duplicate types or just leave this as is here.